python/ray/data/_internal/execution/operators/base_physical_operator.py

```diff
@@ -152,6 +164,9 @@ def close_sub_progress_bars(self):
     def supports_fusion(self):
         return True
 
+    def implements_accurate_memory_accounting(self):
+        return True
```

Comment:
Will remove this method altogether in a follow-up PR. Wanted to keep this PR relatively small